### ISSUE TITLE
[Lab 6] Input para el select

### ISSUE THREAD

**Issue by pfnieto** [PUC-IIC3141/2017-1#12]
En la parte del select, en que momento se supone que debemos entregarle el valor por el cual queremos seleccionar al programa?
Y el filtro de selección solo es necesario soportarlo sobre la columna de los años, cierto?

**Comment by cirojas** [comment]
Sí, el filtro de selección exclusivo para la primera columna y puedes suponer que será un entero.

Tienes que pasar el valor a través de los argumentos de entrada. En tu clase principar (la análoga a `WordCount`), luego de `Configuration conf = new Configuration()` puedes pasarle un parámetro `conf.setInt("c", Integer.parseInt(args[2]))`, luego en el mapper puedes obtener este valor haciendo `Configuration conf = context.getConfiguration();int c = conf.getInt("c", 0);`

**Comment by pfnieto** [comment]
Perfect, con eso funcionó.
Ya tengo el laboratorio listo, está bien si hacemos la corrección el Lunes después del examen?
